app/assets/stylesheets/base.scss

```diff
@@ -47,6 +47,7 @@ h1 {
   color: $sul-h1-font-color;
   font-weight: 300;
   line-height: 1.2em;
+  margin-bottom: 30px;
```

Comment:
@jvine we added this to all `<h1>`s.  Is that okay? (It shows up on the login by pin / reset pin pages as well).

Reply:
Yes, nice.